index.js

```diff
@@ -14,7 +14,8 @@ module.exports = function (ms) {
 		});
 	}
 
-	thunk.then = promise.then.bind(promise);
+        thunk.then = promise.then.bind(promise);
+        thunk.catch = promise.catch.bind(promise);
```

Comment:
I would recommend installing an EditorConfig plugin for your editor, which will fix this for you.

http://editorconfig.org/